crates/router/tests/connectors/wise.rs

```diff
@@ -1 +1,167 @@
+                }),
+                ..Default::default()
+            }),
+            country: Some(api_models::enums::CountryAlpha2::NL),
+            currency: Some(enums::Currency::GBP),
+            ..Default::default()
+        })
+    }
+}
+
+static CONNECTOR: WiseTest = WiseTest {};
+
+/******************** Payouts test cases ********************/
+// Creates a recipient at connector's end
+#[cfg(feature = "payouts")]
+#[actix_web::test]
+async fn should_create_payout_recipient() {
+    let mut payment_info = WiseTest::get_payment_info();
+    payment_info = payment_info.map(|mut pi| {
+        pi.payout_method_data = Some(api::PayoutMethodData::Bank(api::payouts::BankPayout::Bacs(
+            api::BacsBankTransfer {
+                bank_sort_code: "231470".to_string(),
+                bank_account_number: "28821822".to_string(),
+                bank_name: "Deutsche Bank".to_string(),
+                bank_country_code: enums::CountryAlpha2::NL,
+                bank_city: "Amsterdam".to_string(),
+            },
+        )));
+        pi
+    });
+    let response = CONNECTOR
+        .create_payout_recipient(enums::PayoutType::Bank, payment_info)
+        .await
+        .expect("Payout recipient creation response");
+    assert_eq!(
+        response.status.unwrap(),
+        enums::PayoutStatus::RequiresCreation
+    );
+}
+
+// Create BACS payout
+#[cfg(feature = "payouts")]
+#[actix_web::test]
+async fn should_create_bacs_payout() {
+    let mut payment_info = WiseTest::get_payment_info();
+    payment_info = payment_info.map(|mut pi| {
+        pi.payout_method_data = Some(api::PayoutMethodData::Bank(api::payouts::BankPayout::Bacs(
+            api::BacsBankTransfer {
+                bank_sort_code: "231470".to_string(),
+                bank_account_number: "28821822".to_string(),
+                bank_name: "Deutsche Bank".to_string(),
+                bank_country_code: enums::CountryAlpha2::NL,
+                bank_city: "Amsterdam".to_string(),
+            },
+        )));
+        pi
+    });
+    // Create recipient
+    let recipient_res = CONNECTOR
+        .create_payout_recipient(enums::PayoutType::Bank, payment_info.to_owned())
+        .await
+        .expect("Payout recipient response");
+    assert_eq!(
+        recipient_res.status.unwrap(),
+        enums::PayoutStatus::RequiresCreation
+    );
+
+    // Create payout
+    let create_res: types::PayoutsResponseData = CONNECTOR
+        .create_payout(
+            Some(recipient_res.connector_payout_id),
+            enums::PayoutType::Bank,
+            payment_info,
+        )
+        .await
+        .expect("Payout bank creation response");
+    assert_eq!(
+        create_res.status.unwrap(),
+        enums::PayoutStatus::RequiresFulfillment
+    );
+}
+
+// Create and fulfill BACS payout
+#[cfg(feature = "payouts")]
+#[actix_web::test]
+async fn should_create_and_fulfill_bacs_payout() {
+    let mut payment_info = WiseTest::get_payment_info();
```

Comment:
add all your default data for payout within get_payout_info and use it in all other places below
```suggestion
    let mut payout_info = WiseTest::get_payout_info();
```